web/src/user_profile.js

```diff
@@ -97,6 +97,11 @@ export function update_user_profile_streams_list_for_users(user_ids) {
     }
 }
 
+export function update_bot_profile_modal_ui(bot_id) {
+    const bot = people.get_by_user_id(bot_id);
```

Comment:
We can avoid updating the UI here if the bot's profile modal is not opened. And we can do the same for users as well.

Reply:
Done.